add note that kapacitor is optional (#4600)
influxdata/docs-v2: content/chronograf/v1.10/introduction/installation.md

3. Enter a name for your connection.
4. Leave the **Username** and **Password** fields blank unless you've specifically enabled authorization in Kapacitor.
5. Click **Connect**.

{{% note %}}

> **Note:** Using [Kapacitor](https://docs.influxdata.com/kapacitor/v1.6/) is optional.  You do not need to install Kapacitor in order to use Chronograf. 

Reviewer: ```suggestion
**Note:** Using [Kapacitor](/kapacitor/v1.6/) is optional and not required to use Chronograf. 
```
